phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -174,6 +178,11 @@
         "If specified, uses Snapshots for async index building (optional)");
     private static final Option TENANT_ID_OPTION = new Option("tenant", "tenant-id", true,
         "If specified, uses Tenant connection for tenant view index building (optional)");
+
+    private static final Option DELETE_ALL_AND_REBUILD_OPTION = new Option("deleteall", "delete-all-and-rebuild", false,
+            "Applicable only on table indexes, not applicable to view indexes. "
```

Comment:
How about "Applicable only to global indexes on tables, not to local or view indexes"?

Reply:
Much better, will change